usr/bin/nxdk-c++

```diff
@@ -17,6 +17,7 @@ ${CLANG} -target i386-pc-win32 -march=pentium3 \
 -I${NXDK_DIR}/lib/winapi/Winmm \
 -I${NXDK_DIR}/lib/winapi/Ws2_32 \
 -I${NXDK_DIR}/lib/xboxrt/vcruntime \
--Wno-ignored-attributes -DNXDK -D__STDC__=1 \
+-Wno-ignored-attributes -DNXDK -D__XBOX__=1 -D__STDC__=1 \
+-fuse-ld=nxdk-link \
```

Comment:
Untested, but this is meant to enable stuff like:

```
nxdk-cc main.c -o default.exe
```

(Which is also something that autoconf will do internally to verify that the compiler works)